modules/linear_boltzmann_solvers/a_lbs_solver/lbs_structs.h

```diff
@@ -83,9 +83,8 @@ struct BoundaryPreference
   std::string source_function;
 };
 
-enum SourceFlags : int
+enum SourceFlag
```

Comment:
This could also be made into `enum class SourceFlag` to improve type safety, but then all the constants in the code would be referred to as `SourceFlag::ENUM_VALUE`. This may not be needed, since I made `Source` such that it can only be initialized using `SourceFlag`, i.e. code like this `Source src(1233);` is not gonna compile, so there should be enough safety to prevent people to initialize `Source` with random enums/values/etc.

Reply:
Source --> SourceFlags
enum SourceFlag --> SourceType

Reply:
Done